cpp/src/io/parquet/writer_impl.cu

```diff
@@ -656,11 +686,12 @@ std::unique_ptr<std::vector<uint8_t>> writer::impl::write(
   const table_metadata *metadata,
   bool return_filemetadata,
   const std::string &column_chunks_file_path,
-  bool int96_timestamps,
+  std::vector<uint8_t> *decimal_precisions,
   rmm::cuda_stream_view stream)
 {
   pq_chunked_state state{metadata, SingleWriteMode::YES, int96_timestamps, stream};
```

Comment:
I'm a bit lost about the changes related to `int96_timestamps`. It's removed in some APIs, but kept in other places. I also can't track down where the value in this line is coming from. 
Can you please summarize what changes are made to this option?

Reply:
I was trying to find this too. It seems to have been saved from options struct into impl object during construction.

Reply:
So as @devavret said, I found when looking this over that we both capture the value of int96 AND pass it in to the function. This is simply unnecessary and so I removed the duplicated parameter. I had to whiteboard all this option stuff out to wrap my head around it and it's ripe for refactoring for simplification.